opsworks_custom_cookbooks/recipes/iptables.rb

```diff
@@ -0,0 +1,40 @@
+subnet_cidr = node[:deploy]['aviary'][:environment_variables][:SUBNET_CIDR]
+
+bash "configure_iptables" do
+  code <<-EOH
+    iptables -P INPUT ACCEPT
+    iptables -P FORWARD ACCEPT
+    iptables -P OUTPUT ACCEPT
+    iptables -F
+
+    iptables -P INPUT DROP
+    iptables -P FORWARD DROP
+    iptables -P OUTPUT DROP
+
+    iptables -A INPUT -i eth0 -p tcp --dport 22 -m state --state NEW,ESTABLISHED -j ACCEPT
+    iptables -A OUTPUT -o eth0 -p tcp --sport 22 -m state --state ESTABLISHED -j ACCEPT
+
+    iptables -A INPUT -i lo -j ACCEPT
+    iptables -A OUTPUT -o lo -j ACCEPT
+    iptables -A INPUT -i eth0 -p tcp --dport 80 -m state --state NEW,ESTABLISHED -j ACCEPT
```

Comment:
Those were added when we troubleshooted the chef connectivity. As I read, chef using ports 80 and 443 to communicate with nodes. If those ports are not needed, I can remove them, I would prefer to test it first.